packages/medusa/src/repositories/staged-job.ts

```diff
@@ -1,5 +1,37 @@
 import { EntityRepository, Repository } from "typeorm"
-import { StagedJob } from "../models/staged-job"
+import { StagedJob } from "../models"
+import { QueryDeepPartialEntity } from "typeorm/query-builder/QueryPartialEntity"
+import { RawSqlResultsToEntityTransformer } from "typeorm/query-builder/transformer/RawSqlResultsToEntityTransformer"
+import { RelationIdLoader } from "typeorm/query-builder/relation-id/RelationIdLoader"
 
 @EntityRepository(StagedJob)
-export class StagedJobRepository extends Repository<StagedJob> {}
+export class StagedJobRepository extends Repository<StagedJob> {
+  async insertBulk(jobToCreates: QueryDeepPartialEntity<StagedJob>[]) {
```

Comment:
No otherwise it creates a query per insert which crates very bad performances in the end. 
example:
- 1 query to insert 10ms
- 50 queries with 2ms per query = 100ms

It is not real number but just for the example ahah

Reply:
Ah alright thought TypeORM did bulk inserts when using `save` with an array of entities